bzt/modules/blazemeter.py

```diff
@@ -1105,7 +1344,7 @@ def get_labels(self, master_id):
 
     def get_available_locations(self):
         user_info = self.get_user_info()
-        return {str(x['id']): x for x in user_info['locations'] if not x['id'].startswith('harbor-')}
```

Comment:
Good point. Updated the PR.

Current state: `bzt -locations` shows all locations (including harbors), but Taurus allows to use harbors only with collection-based tests.